packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -53,8 +38,10 @@ def test_if_marathon_app_can_be_debugged(dcos_api_session):
     app_id = 'integration-test-{}'.format(test_uuid)
     with dcos_api_session.marathon.deploy_and_cleanup(app):
         # Fetch the mesos master state once the task is running
-        master_state_url = 'http://{}:{}/state'.format(dcos_api_session.masters[0], 5050)
-        r = requests.get(master_state_url)
+        scheme = dcos_api_session.default_url.scheme
+        master = dcos_api_session.masters[0]
```

Comment:
Good catch. I've renamed it to `master_ip` and use it in the `dcos_api_session.get(...)` on the next two lines instead of calling `dcos_api_session.masters[0]` again.